<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab6DataBase</class>
 <widget class="QWidget" name="Tab6DataBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="pDateTimeEditFrom">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2022</year>
         <month>6</month>
         <day>10</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="pDateTimeEditTo">
       <property name="dateTime">
        <datetime>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2022</year>
         <month>6</month>
         <day>30</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBsearchButton">
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBdeleteButton">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pTableChartHoriLayout">
     <item>
      <widget class="QTableWidget" name="pTableWidget">
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>110</number>
       </attribute>
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>cds</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>temp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>humi</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
